docs/source/Activities/Architecture_Review_Methodology.md

```diff
@@ -1,6 +1,6 @@
 # Architecture Review Methodology
 
-The RABET-V Architecture Review is designed to evaluate the solution’s architectural support for the [RABET-V security control families](/Overview/Security_Control_Family.md). This evaluation produces an architectural maturity score for each security control family and identifies the components that provide each [security service](/Appendices/RABET-V_Glossary.md#security-service). This score does not measure how well the [product](/Appendices/RABET-V_Glossary.md#product) executes the security service (i.e., its capability level), just how mature the architecture is that supports the security service. The [Security Services Capability Maturity](/Security_Services_Capability_Maturity_Index/README.md) level is a separate metric determined in the [Security Claims Validation](/Activities/Security_Claims_Validation.md) that indicates how well the product provides the security services.
+The RABET-V Architecture Review is designed to evaluate the solution’s architectural support for the [RABET-V security control families](/Overview/Security_Control_Family.md). This evaluation produces an architectural maturity score for each security control family and identifies the components that provide each {term}`security service`. This score does not measure how well the {term}`product` executes the security service (i.e., its capability level), just how mature the architecture is that supports the security service. The [Security Services Capability Maturity](/Security_Services_Capability_Maturity_Index/README.md) level is a separate metric determined in the [Security Claims Validation](/Activities/Security_Claims_Validation.md) that indicates how well the product provides the security services.
```

Comment:
@JDziurlaj Since I updated the Markdown parser from recommonmark to [MyST](https://myst-parser.readthedocs.io/en/latest/index.html) (NB: recommonmark isn't supported anymore) I made a few edits that allow us to use some of Sphinx's Glossary features (i.e., specifying that something is a "term" to be defined in the glossary and Sphinx flags it if it can't find a matching term in the glossary).